docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
       POSTGRES_USER: sequelize_test
       POSTGRES_PASSWORD: sequelize_test
       POSTGRES_DB: sequelize_test
+      PGDATA: /dev/shm/pgdata/data
```

Comment:
I think you need to mount ramdisk first ? For Circle CI postgres image its pre mounted https://circleci.com/docs/2.0/postgres-config/#optimizing-postgres-images

But here we use different image as we need PostGIS support